tools/ci_build/requirements-transformers-test.txt

```diff
@@ -3,7 +3,8 @@ packaging
 protobuf==3.20.2
 numpy==1.24.0 ; python_version < '3.12'
 numpy==1.26.0 ; python_version >= '3.12'
+torch
```

Comment:
Note that this only works in Linux.
For windows, it will install CPU only torch. So it might not work for Windows pipeline.
It is fine right now since we only enable the test in Linux.